### PR TITLE
Improve state explanation

### PR DESCRIPTION
**Pull request by orkunkl**
Resolves https://github.com/iov-one/docs/issues/90

### CODE REVIEW

File: docs/weave-tutorial/02-domain.md [hunk 1]
```diff
@@ -6,22 +6,24 @@ sidebar_label: Domain
 
 The first thing we consider is the data we want to store, which is known as the state. After that, we can focus on the messages, which trigger state transitions. All blockchain state must be stored in our Merkle-ized store, which provides validity hashes and proofs. This is exposed to the application as a basic key-value store, which also allows in-order iteration over the keys. On top of this, we have built some tools, such as secondary indexes and sequences, similar to how [Storm adds an ORM](https://github.com/asdine/storm#simple-crud-system) on top of [BoltDB’s key-value store](https://github.com/boltdb/bolt#using-buckets). We have avoided struct tags and tried to type as strictly as we can (without using generics).
```
Comment (JimIOV) [inline docs/weave-tutorial/02-domain.md]:
```suggestion
The first thing we consider is the data we want to store, which is known as the state. After that, we can focus on the messages, which trigger state transitions. All blockchain state must be stored in our Merkle-ized store, which provides validity hashes and proofs. This is exposed to the application as a basic key-value store, which also allows in-order iteration over the keys. On top of this we have built some tools, such as secondary indexes and sequences, similar to how [Storm adds an ORM](https://github.com/asdine/storm#simple-crud-system) on top of [BoltDB’s key-value store](https://github.com/boltdb/bolt#using-buckets). We have avoided struct tags and tried to type as strictly as we can (without using generics).
```

**Comment by orkunkl** [comment]
Blockchain may have multiple `states` that it keeps, but `states` will collide into Blockchain `state` if you consider all.

File: docs/weave-tutorial/02-domain.md [hunk 2]
```diff
@@ -6,22 +6,24 @@ sidebar_label: Domain
...
 
-We are building an Order book application. This topic is quite hot at the time of writing this tutorial.
+## State
 
-- **Markets** will contain rules as to who (which public keys) may create an **order book** or who can cancel an order. The blockchain may have multiple markets, and each market may have multiple Orderbooks; however, each token pair can only have one order book per market.
+State as in [Turing machine](https://en.wikipedia.org/wiki/Turing_machine), is how the system's data is presented this contained. We are building an Order book application to solidify the state concept. Exchange topic is quite hot at the time of writing this tutorial.
```
Comment (JimIOV) [inline docs/weave-tutorial/02-domain.md]: I don't understand hw this first sentence ends: 'this contained' 

Looks like a typo?

File: docs/weave-tutorial/02-domain.md [hunk 3]
```diff
@@ -6,22 +6,24 @@ sidebar_label: Domain
...
+
+- **Markets** will contain rules as to who (which public keys) may create an **order book** or who can cancel an order. The term _market_ here, defines the state of an entity that represents the real markets. The blockchain application may have multiple markets, and each market may have multiple Orderbooks; however, each token pair can only have one order book per market.
```
Comment (JimIOV) [inline docs/weave-tutorial/02-domain.md]:
```suggestion
- **Markets** will contain rules as to who (which public keys) may create an **order book** or who can cancel an order. The term _market_ here defines the state of an entity that represents the real markets. The blockchain application may have multiple markets, and each market may have multiple Orderbooks; however, each token pair can only have one order book per market.
```

File: docs/weave-tutorial/02-domain.md [hunk 4]
```diff
@@ -6,22 +6,24 @@ sidebar_label: Domain
...
 There is no global chain owner, but each market has its own chain owner that adds Order books and sets fees. This could be one person, a multisig, or a governance contract (DAO). When adding an order book, the market with a given market ID is checked, then looks up the owner _for that market_. Rather than having one owner that can create Order books for all markets, each market stores the information on which owner can update it.
```
Comment (JimIOV) [inline docs/weave-tutorial/02-domain.md]: What does DAO stand for?

File: docs/weave-tutorial/02-domain.md [hunk 5]
```diff
@@ -6,22 +6,24 @@ sidebar_label: Domain
...
 
-- **Order books** will define which Ask/Bid market it holds and the number of available orders. Order book will contain orders and trades.
+- **Order book** state will defines which Ask/Bid market it holds and the number of available orders. Order book will contain orders and trades.
```
Comment (JimIOV) [inline docs/weave-tutorial/02-domain.md]:
```suggestion
- **Order book** state will define which Ask/Bid market it holds and the number of available orders. An Order book will contain orders and trades.
```

**Pull request merged**